src/System.CommandLine/Argument{T}.cs

```diff
@@ -68,7 +68,7 @@ public Argument(Func<T> getDefaultValue) : this()
                 }
                 else
                 {
-                    value = default(T);
+                    value = default(T)!;
```

Comment:
This depends on the specific version of the compiler used. This suppression should no longer be necessary (as of Roslyn 3.6 I suspect, could double-check)